### ISSUE TITLE
`Part.get_category_parameter_templates` fails to return parameters

### ISSUE THREAD

**Issue by eeintech** [inventree/inventree-python#131]
Step to reproduce:
1. Add category parameters through InvenTree UI (Settings > Categories)
2. Navigate to `<inventree-url>/api/part/category/<category-id>/parameters/`
3. There should be a list of parameters associated to category, currently they are missing

The `fetch_parent` option should also be tested when fixed.

**Comment by SchrodingersGat** [comment]
@eeintech the URL format is wrong. It should be something like:

```
http://localhost:8000/api/part/category/parameters/?category=82
```

**Comment by SchrodingersGat** [comment]
@eeintech fixed, and `0.8.1` is now available via pypi

**Comment by eeintech** [comment]
@SchrodingersGat Lightning fast fix! Amazing! Thanks a lot :smiley:
